site/content/courses/12-001-introduction-to-geology-fall-2013/_index.md

```diff
@@ -4,6 +4,17 @@ course_id: 12-001-introduction-to-geology-fall-2013
 course_title: Introduction to Geology
 course_image_url: >-
   https://open-learning-course-data.s3.amazonaws.com/12-001-introduction-to-geology-fall-2013/76ea2d54123485055dd1e6621b9edf6a_12-001f13.jpg
+course_image_alternate_text: >-
+  A map of North America with colors that represent the ages of the underlying
+  rocks.
+course_image_caption_text: >-
+  <p>The North America Tapestry of Time and Terrain. This map is a portrait of
+  the land surface of North America and the ages of its underlying rock
+  formations. This digital tapestry outlines the geologic story of continental
+  collision and break-up, mountain-building, river erosion and deposition,
+  ice-cap glaciation, volcanism, and other events and processes that have shaped
+  the region over the last 2.6 billion years. (Image and text courtesy of the <a
+  href="http://www.usgs.gov/">U.S. Geological Survey</a>.)</p>
```

Comment:
in the long run we might want to try to convert this html to markdown, what do you think? for now I think this is ok but maybe we should file an issue?

Reply:
Yea, would be a good idea to create a ticket to go through all the fields we're pulling in that aren't the `text` field from `course_pages` like the course description and image caption and convert them to markdown before writing to the course home page's front matter.  I anticipate there might be some formatting issues there with line breaks, since it would be in a front matter key and not the document body.  Definitely worth exploring.

Reply:
https://github.com/mitodl/ocw-to-hugo/issues/31